apply/apply.go

```diff
@@ -219,16 +219,26 @@ func applyTFE(fs afero.Fs, plan *plan.Plan, tmpl *templates.T) error {
 	}
 
 	logrus.Debug("applying tfe")
-	path := fmt.Sprintf("%s/tfe", rootPath)
+	path := filepath.Join(rootPath, "tfe")
 	err := fs.MkdirAll(path, 0755)
 	if err != nil {
 		return errors.Wrapf(err, "unable to make directory %s", path)
 	}
-	err = applyTree(fs, tmpl.Components[v2.ComponentKindTerraform], tmpl.Common, path, plan.TFE)
+
+	// only create the locals.tf.json template is one doesn't exist yet
+	tfePath := filepath.Join(rootPath, "tfe", "locals.tf.json")
```

Comment:
could join with `path` since that already joined the first 2 items:
```suggestion
	// only create the locals.tf.json template if one doesn't exist yet
	tfePath := filepath.Join(path, "locals.tf.json")
```